src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -235,20 +235,16 @@ public function output(
      * exists => a temporary one is created.
      *
      * @param object $object
+     * @param FieldDescriptionInterface $fieldDescription
      *
      * @throws \RuntimeException
      *
      * @return mixed
      */
     public function getValueFromFieldDescription(
         $object,
-        FieldDescriptionInterface $fieldDescription,
-        array $params = []
```

Comment:
These changes seems to be BC-break to me.

Should we add 
```
NEXT_MAJOR: Remove the loop check and the third param
```
And add a deprecation if the $params args is not empty. (Passing a third argument is deprecated and will be ignored in 4.0).

Reply:
Formally, yes, but I don't think this method is ever called outside this class. This class should only be used using twig functions and filters and no of those call getValueFromFieldDescription directly.

Reply:
Unfortunaly, it's not because it should not be called that it's not called.
We have to respect our BC-promise.

In the next major we can change the visibility to avoid this issue.

Reply:
OK!